src/csharp/Grpc.Core/Interceptors/ChannelExtensions.cs

```diff
@@ -0,0 +1,65 @@
+#region Copyright notice and license
+
+// Copyright 2018 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+
+namespace Grpc.Core.Interceptors
+{
+    /// <summary>
+    /// Provides extension methods to make it easy to register interceptors on Channel objects.
+    /// This is an EXPERIMENTAL API.
+    /// </summary>
+    public static class ChannelExtensions
```

Comment:
something to consider: 
if these extensions are under Grpc.Core.Interceptors namespace, then the extension methods won't be visible to code completion unless you  add `using Grpc.Core.Interceptor;`  to the source. Not sure if that's desired or not.

Reply:
That is intended indeed, but let me know what is preferable to you. I am 60-40 in favor of the current organization.